### ISSUE TITLE
Cube Support

### ISSUE THREAD

**Issue by jquinones-eti** [evidence-dev/evidence#1308]
## What do you use this database for?

Headless semantic layer for data web apps. Allows for a single data model to be used between multiple BI tools.

## Are there any popular and well-maintained clients for this database?

[@cubejs-client/core](https://www.npmjs.com/package/@cubejs-client/core)

## Are you interested in contributing this database connector?

I do not currently have the time to contribute.

[https://cube.dev/](https://cube.dev/)

**Comment by ItsMeBrianD** [comment]
Hey @jquinones-eti!

I took a look into how we could integrate with Cube and I think the quickest answer is with the [SQL API](https://cube.dev/docs/product/apis-integrations/sql-api). This will probably work with the Postgres connector that is built into Evidence.

**Comment by archiewood** [comment]
I tried out cube a year ago with Evidence and the SQL API (using Evidence's Postgres connector), and it mainly worked.

There was a cube bug at the time when you wrote invalid SQL, but the cube team [say that it has been fixed](https://github.com/cube-js/cube/issues/5095). I haven't tested it since then

**Comment by archiewood** [comment]
Glad to hear it - was there any specific config you needed to get it going? 
Happy to add an entry for Cube.js to the docs

**Comment by jquinones-eti** [comment]
Nope. Just had to enable the SQL API in Cube (disabled by default) and then use the provided credentials to fill in the PostgreSQL connection parameters in Evidence.

**Comment by archiewood** [comment]
I've added Cube to our docs, based on the above - shout if you think we need to make any updates.

https://docs.evidence.dev/core-concepts/data-sources/#cube
